cmd/res/configuration.toml

```diff
@@ -2,106 +2,19 @@ MaxEventSize = 0 # value 0 unlimit the maximum event size that can be sent to me
-    #Gateway="my-iot-gateway" # Tag must be added here since Env Override can only change existing value, not added new ones.
 
 [Service]
-HealthCheckInterval = "10s"
-Host = "localhost"
-ServerBindAddr = ""  # blank value defaults to Service.Host value
 Port = 59900
 StartupMsg = "device virtual started"
-# MaxRequestSize limit the request body size in byte of put command
-MaxRequestSize = 0 # value 0 unlimited request size.
-RequestTimeout = "5s"
-  [Service.CORSConfiguration]
-  EnableCORS = false
-  CORSAllowCredentials = false
-  CORSAllowedOrigin = "https://localhost"
-  CORSAllowedMethods = "GET, POST, PUT, PATCH, DELETE"
-  CORSAllowedHeaders = "Authorization, Accept, Accept-Language, Content-Language, Content-Type, X-Correlation-ID"
-  CORSExposeHeaders = "Cache-Control, Content-Language, Content-Length, Content-Type, Expires, Last-Modified, Pragma, X-Correlation-ID"
-  CORSMaxAge = 3600
-
-[Registry]
-Host = "localhost"
-Port = 8500
-Type = "consul"
-
-[Clients]
-  [Clients.core-data]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59880
-
-  [Clients.core-metadata]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59881
 
 [MessageBus]
-Protocol = "redis"
-Host = "localhost"
-Port = 6379
-Type = "redis"
-AuthMode = "usernamepassword"  # required for redis messagebus (secure or insecure).
-SecretName = "redisdb"
   [MessageBus.Topics]
   PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name>/<source-name> will be added to this Publish Topic prefix
   DeviceCommandRequestTopic = "edgex/device/command/request/device-virtual/#"  # subscribing for inbound command requests
   ResponseTopicPrefix = "edgex/response" # /<service-name>/<request-id> will be added to this prefix topic
   MetadataSystemEventTopic = "edgex/system-events/core-metadata/+/+/device-virtual/#"
   [MessageBus.Optional]
-  # Default MQTT & NATS Specific options that need to be here to enable environment variable overrides of them
   ClientId = "device-virtual"
-  Qos =  "0" # Quality of Service values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
-  KeepAlive =  "10" # Seconds (must be 2 or greater)
-  Retained = "false"
-  AutoReconnect = "true"
-  ConnectTimeout = "5" # Seconds
-  SkipCertVerify = "false"
-  # Default NATS Specific options that need to be here to enable environment variable overrides of them
-  Format = "nats"
-  RetryOnFailedConnect = "true"
-  QueueGroup = ""
-  Durable = ""
-  AutoProvision = "true"
-  Deliver = "new"
-  DefaultPubRetryAttempts = "2"
-  Subject = "edgex/#" # Required for NATS JetStream only for stream auto provisioning
 
 [Device]
-DataTransform = true
-MaxCmdOps = 128
-MaxCmdValueLen = 256
-ProfilesDir = "./res/profiles"
-DevicesDir = "./res/devices"
-UpdateLastConnected = false
-EnableAsyncReadings = true
-AsyncBufferSize = 16
-Labels = []
-UseMessageBus = true
-  [Device.Discovery]
-  Enabled = false
-  Interval = "30s"
+  ProvisionWatchersDir = ""
```

Comment:
No longer needed with the tweak we made to common.

Reply:
fixed